gui/README.md

```diff
@@ -64,45 +64,6 @@ To run the development server:
 npm run tauri dev
 ```
 
-## Getting started for developement - macOS
```

Comment:
Why was this removed? I'm not even necessarily opposed to it, just wondering why.